dapp/src/components/ConnectWalletModal/ConnectWalletButton.tsx

```diff
@@ -54,13 +55,15 @@ export default function ConnectWalletButton({
   isSelected,
   connector,
   onSuccess,
+  isReconnecting,
 }: ConnectWalletButtonProps) {
   const { isEmbed } = useIsEmbed()
   const {
     address,
     onConnect,
     onDisconnect,
     status: connectionStatus,
+    reconnectStatus,
```

Comment:
Non-blocking comment

The `onConnect` function works in normal mode and in reconnecting mode. So I think we don't need to return two independent statuses but just one.

Reply:
aecbe9a